hal/ia32/exceptions.c

```diff
@@ -158,6 +158,11 @@ static void exceptions_defaultHandler(unsigned int n, exc_context_t *ctx)
 	hal_consolePrint(buff);
 	hal_consolePrint("\033[0m");
 	hal_interruptsDisableAll();
+
+#ifdef NDEBUG
+	hal_cpuReboot();
+#endif
+
 	hal_cpuHalt();
```

Comment:
For safety `hal_cpuHalt();` may be wrapped in an endless loop like other targets.